ERCS/erc-7717.md

```diff
@@ -0,0 +1,250 @@
+---
+eip: 7717
+title: Deterministic AA wallet
+description: Propose a non-custodial interface for service providers to provide AA accounts.
```

Comment:
```suggestion
description: Non-custodial interface for service providers to provide AA accounts.
```

Cut out a bit of fluff, but I think you should expand on who "service providers" are, and what "provide" means in this context.